Octokit.Tests.Integration/Clients/PullRequestReviewCommentReactionsClientTests.cs

```diff
@@ -40,15 +40,42 @@ public PullRequestReviewCommentReactionsClientTests()
 
         AssertComment(commentFromGitHub, body, position);
 
-        var pullRequestReviewCommentReaction = await _github.Reaction.PullRequestReviewComment.Create(_context.RepositoryOwner, _context.RepositoryName, commentFromGitHub.Id, new NewReaction(ReactionType.Heart));
-
-        Assert.NotNull(pullRequestReviewCommentReaction);
-
-        Assert.IsType<Reaction>(pullRequestReviewCommentReaction);
-
-        Assert.Equal(ReactionType.Heart, pullRequestReviewCommentReaction.Content);
-
-        Assert.Equal(commentFromGitHub.User.Id, pullRequestReviewCommentReaction.User.Id);
+        var newReactionConfused = new NewReaction(ReactionType.Confused);
+        var newReactionHeart = new NewReaction(ReactionType.Heart);
```

Comment:
Although I definitely want integration tests to try every enum value, I dont think we need to be this verbose (plus in the future if more reactions are added, we'd need to remember to add them to this test).

So whatabout soething like this:

``` csharp
foreach (ReactionType reactionType in Enum.GetValues(typeof(ReactionType)))
{
    var newReaction = new NewReaction(reactionType);

    var reaction = await _github.Reaction.CommitComment.Create(_context.RepositoryOwner, _context.RepositoryName, result.Id, newReaction);

    Assert.IsType<Reaction>(reaction);
    Assert.Equal(reactionType, reaction.Content);
    Assert.Equal(result.User.Id, reaction.User.Id);
}
```